packages/fuel-gauge/test/fixtures/forc-projects/advanced-logging-other-contract/src/main.sw

```diff
@@ -4,6 +4,38 @@ use std::logging::log;
 
 use advanced_logging_other_contract_abi::AdvancedLoggingOtherContract;
 
+impl AbiEncode for str[25] {
+    fn abi_encode(self, ref mut buffer: Buffer) {
+        let s = from_str_array(self);
+
+        let len = s.len();
+        let ptr = s.as_ptr();
+
+        let mut i = 0;
+        while i < len {
+            let byte = ptr.add::<u8>(i).read::<u8>();
+            buffer.push(byte);
+            i += 1;
+        }
+    }
+}
+
+impl AbiEncode for str[34] {
```

Comment:
cc @FuelLabs/sdk-ts I could change the tests so that they all use a `str` length less than or equal to 5. These lengths do not require custom implementations? The tests are still valid with shorter strings.